net/nimble/host/mesh/src/net.c

```diff
@@ -556,40 +556,57 @@ void bt_mesh_iv_update(u32_t iv_index, bool iv_update)
 {
 	int i;
 
-	if (iv_index < bt_mesh.iv_index || iv_index > bt_mesh.iv_index + 42) {
-		BT_ERR("IV Index completely out of sync: 0x%08x != 0x%08x",
-			iv_index, bt_mesh.iv_index);
-		return;
-	}
+	/* If we are in normal*/
+	if (!bt_mesh.iv_update) {
```

Comment:
Looks fine to me, but I'd reverse the if-statement to avoid having to do a negative, i.e. the first branch would be if (bt_mesh.iv_update), which is actually what the code already has from before. I've got another pending patch related to Friend support that modifies bt_mesh_iv_update() to return a bool to indicate if the state was changed or not, so I'll be doing the Zephyr port of this patch on top of my change.